spot-ml/ml_ops.sh

```diff
@@ -103,20 +96,11 @@ time spark-submit --class "org.apache.spot.SuspiciousConnects" \
   --input ${RAWDATA_PATH}  \
   --dupfactor ${DUPFACTOR} \
   --feedback ${FEEDBACK_PATH} \
-  --model ${LPATH}/model.dat \
-  --topicdoc ${LPATH}/final.gamma \
-  --topicword ${LPATH}/final.beta \
-  --lpath ${LPATH} \
-  --ldapath ${LDAPATH} \
-  --luser ${LUSER} \
-  --mpicmd ${MPI_CMD}  \
-  --proccount ${PROCESS_COUNT} \
   --topiccount ${TOPIC_COUNT} \
```

Comment:
that seems reasonable for the CLI argument names, internal variable names camelCased of course